.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Please help potential reviewers to understand this pull request and speed up 
+the process by answering the following questions, where appropriate.
+
+# Why was this pull request created?
+
+# How is this achieved?
```

Comment:
Obviously the goal of this section is to give a TL;DR technical overview of what the commits do.